core/src/test/java/hudson/util/ProcessTreeTest.java

```diff
@@ -27,8 +29,7 @@
     
     @Test public void remoting() throws Exception {
         // on some platforms where we fail to list any processes, this test will just not work
-        if (ProcessTree.get()==ProcessTree.DEFAULT)
-            return;
+        Assume.assumeTrue(ProcessTree.get()!=ProcessTree.DEFAULT);
```

Comment:
I find `assumeThat(ProcessTree.get(), not(sameInstance(ProcessTree.DEFAULT)))` is more readable that a true followed by a negative